src/utils/index.ts

```diff
@@ -92,3 +92,21 @@ export const getBlsGameInfos = ({
         transactionResult
     });
 };
+
+export const getBlsGameInfosWithDraw = ({
+    coinType,
+    corePackageId,
+    gameSeed,
+    moduleName,
+    structName,
+    transactionResult
+}: BlsGameInfosInput): GameInfo[] => {
+    // UNIHOUSE_PACKAGE - BetDataWithDraw did not exist until unihouse v4
+    const filterString = `${UNIHOUSE_PACKAGE}::${BLS_SETTLER_MODULE_NAME}::BetDataWithDraw<${coinType}, ${corePackageId}::${moduleName}::${structName}>`;
```

Comment:
@healthyred i needed to use the v4 unihouse package id in order to grab the game info and parse the settlement.

this is different than the others and i would like to get your blessing on this. you had mentioned previously that the way i wanted to generalize this kind of behavior wouldn't work, and perhaps this is part of it.